### PR TITLE
Cleanup core apps

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: apps/comments/appinfo/info.xml [hunk 1]
```diff
@@ -10,7 +10,4 @@
 	<dependencies>
 		<owncloud min-version="9.0" max-version="9.0" />
 	</dependencies>
-	<documentation>
-		<user>user-comments</user>
```
Comment (nickvergessen) [inline apps/comments/appinfo/info.xml]: Entry is not valid, there are no docs for "comments", neither is there an entry in https://github.com/owncloud/documentation/blob/f8fec408eff462a3c2c06b5c7e8f9cdfb8825b02/go.php#L33 which would make the link work

File: apps/systemtags/appinfo/info.xml [hunk 2]
```diff
@@ -10,9 +10,6 @@
 	<dependencies>
 		<owncloud min-version="9.0" max-version="9.0" />
 	</dependencies>
-	<documentation>
-		<user>user-systemtags</user>
```
Comment (nickvergessen) [inline apps/systemtags/appinfo/info.xml]: Entry is not valid, there are no docs for "systemtags", neither is there an entry in https://github.com/owncloud/documentation/blob/f8fec408eff462a3c2c06b5c7e8f9cdfb8825b02/go.php#L33 which would make the link work

File: tests/lib/appframework/AppTest.php [hunk 3]
```diff
@@ -79,9 +79,9 @@ protected function setUp() {
 		$this->container['OCP\\AppFramework\\Http\\IOutput'] = $this->io;
 		$this->container['urlParams'] = array();
 
-		$this->appPath = __DIR__ . '/../../../apps/namespacetestapp/appinfo';
-		$infoXmlPath = $this->appPath . '/info.xml';
-		mkdir($this->appPath, 0777, true);
+		$this->appPath = __DIR__ . '/../../../apps/namespacetestapp';
```
Comment (nickvergessen) [inline tests/lib/appframework/AppTest.php]: `namespacetestapp` was previously always left behind when running unit tests

File: apps/comments/appinfo/info.xml (hunk 1 above)
Comment (rullzer) [inline apps/comments/appinfo/info.xml]: maybe there should be ;)
but that is a different issue.

**Comment by rullzer** [comment]
:+1:

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**